lib/ui/plugins/callback_cache.cc

```diff
@@ -130,8 +130,6 @@ void DartCallbackCache::LoadCacheFromDisk() {
   Document d;
   d.Parse(cache_contents.c_str());
   if (d.HasParseError() || !d.IsArray()) {
-    FML_LOG(INFO) << "Could not parse callback cache, aborting restore";
```

Comment:
Sorry to nit on such a small change, but the log was documenting what was happening here.  Can you change this to a comment?
```c++
    // Could not parse callback cache, aborting restore.
    return;
```